backend/src/controllers/contracts.js

```diff
@@ -1,6 +1,23 @@
 const useController = require("./useController/index.js");
 const model = require("../models/contracts.js");
+const subcontractorsModel = require("../models/subcontractors");
 const what = { single: "contract", plural: "contracts" };
 const controller = useController(model, what);
 
+controller.getOne = async (request, reply) => {
```

Comment:
I feel like we shouldn't be using getOne here, although it is the same, this could be confusing to the internal getOne of the useController.  if we are going to override this function we should not call it the same thing.  maybe getOneWithSubData or something, that way when we are looking at the routing, we know it is different.

Reply:
In my mind it makes sense to override the existing method from useController since we don't really want to be using the base getOne() because it doesn't come with the necessary data. Leaving the base getOne() not overridden allows devs to use that function even though it doesn't actually do what we want it to. I can change its name to getOneWithSubData() but in my opinion that actually obfuscates what's going on more than overriding does.